pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -665,7 +666,8 @@ func (m *kubeGenericRuntimeManager) computePodResizeAction(pod *v1.Pod, containe
 func (m *kubeGenericRuntimeManager) doPodResizeAction(pod *v1.Pod, podStatus *kubecontainer.PodStatus, podContainerChanges podActions, result kubecontainer.PodSyncResult) {
 	pcm := m.containerManager.NewPodContainerManager()
 	//TODO(vinaykul,InPlacePodVerticalScaling): Figure out best way to get enforceMemoryQoS value (parameter #4 below) in platform-agnostic way
-	podResources := cm.ResourceConfigForPod(pod, m.cpuCFSQuota, uint64((m.cpuCFSQuotaPeriod.Duration)/time.Microsecond), false)
+	cpuExperimentalManagerPolicyStatic := m.containerManager.GetNodeConfig().CPUManagerPolicy == string(cpumanager.PolicyStatic)
```

Comment:
I can see the intent, but cpumanager is GA, so adding back "Experimental" doesn't seem the best way forward to me; I'd prefer something like a feature gate to guard this fix (or any other alternate implementation). I asked sig-arch for input about adding feature gates in this context.

Reply:
cool, lmk happy to swap to whatever verbiage they choose